src/monitor/version_compat.h

```diff
@@ -14,7 +14,12 @@
 
 #include "postgres.h"
 
-#if (PG_VERSION_NUM >= 90600 && PG_VERSION_NUM < 110000)
+/* we only support PG10 and PG11, PG 12 is not supported yet*/
+#if (PG_VERSION_NUM < 100000 || PG_VERSION_NUM >= 120000)
+#error "Unknown or unsupported postgresql version"
+#endif
+
+#if (PG_VERSION_NUM < 110000)
```

Comment:
It would be easier to read now if that was

``` c
#if (PG_VERSION_NUM == 100000)
```

Reply:
minor version numbers may interfere. I saw PG 11.2 version number as 110002 so exact match might cause issues

Reply:
perhaps PG_MAJORVERSION but it is a string.

Reply:
oh, ok then, good point!